components/layout/display_list_builder.rs

```diff
@@ -2124,7 +2124,7 @@ impl FragmentDisplayListBuilding for Fragment {
         // to back).
 
         // Shadows
-        for shadow in text_shadows.iter().rev() {
+        for shadow in text_shadows.iter() {
```

Comment:
Yeah it ended up being easier for the webrender backend to handle shadows in reverse (since the shadow draws trigger on pop), so we just made the API work that way (which is easier for our clients anyway).